azure/v14.1.7/README.md

```diff
@@ -0,0 +1,45 @@
+# :zap: Giant Swarm Release v14.1.7 for Azure :zap:
+
+<< Add description here >>
+
+## Change details
+
+
+### azure-operator [5.5.4](https://github.com/giantswarm/azure-operator/releases/tag/v5.5.4)
+
+Not found
```

Comment:
We have to change this with something else.

Reply:
```suggestion
Meta release for guaranteed node rolling. No changes in functionality.
```
Something like this or should I drop it altogether?

Reply:
No better leave it, LGTM